content/committees/specification/tckprocess/index.md

```diff
@@ -241,6 +249,9 @@ compatible with the testing framework in use so that as the excludes are
 updated, the affected tests are automatically removed from the test
 suite.
 
+###  Adding excluded tests
+Excluded tests should be added back in for every major Jakarta EE release by emptying the test exclude list for every Specification developed in the respective major EE release.   
```

Comment:
Define "major".  Wouldn't we also expect that the exclude lists get emptied on a minor (x.y) release?  We would not expect this for a service (x.y.z) release, but I think we should expect it for both major and minor releases.

Reply:
Some possible options:
1. Only clear the excluded test list for specifications that are being updated/developed for a particular Jakarta EE release.  The burden is on the EE implementation teams to challenge any of those tests that aren't fixed (which slows adoption of that new EE version.)
2. Always clear the excluded test list for specifications that are included in any Jakarta EE minor/major release but not service (x.y.z) releases.

Would it make sense to remove all tests that are repeatedly challenged but not fixed for two consecutive EE major/minor releases?